pkg/proxy/userspace/proxier.go

```diff
@@ -687,12 +696,16 @@ func iptablesCommonPortalArgs(destIP net.IP, destPort int, protocol api.Protocol
 		args = append(args, "-d", fmt.Sprintf("%s/32", destIP.String()))
 	}
 
+	if excludeContainerSourceTraffic {
+		args = append(args, "-m", "physdev", "--physdev-is-in")
```

Comment:
This is not a filter rule at all, it's a nat rule.  You want to match not-bridged packets, which sends them to the proxier.  Bridged packets will go through the normal packet path.   I feel like I am missing something - how would this work at all as it is?